### DISCUSSION TITLE
error to use the application

### DISCUSSION THREAD

**Discussion by littlefury08** [rocksdanister/lively#1752] [General]
**description of the problem**
When I want to enter the application it doesn't let me since it says it's running and when I try to delete it it tells me that it can't since it's running and I don't know what to do to fix it 
I attached an image but it is in Spanish 
![image](https://github.com/rocksdanister/lively/assets/135916838/2eab8e66-eff8-4786-807c-5960753a264a)

**Comment by rocksdanister** [comment]
To stop Lively select Exit from the system-tray menu: https://github.com/rocksdanister/lively/wiki/Getting-Started
Steps to uninstall: https://github.com/rocksdanister/lively/discussions/251
